CommunityFridgeMapApi/tests/unit/test_tag_is_valid.py

```diff
@@ -0,0 +1,25 @@
+from multiprocessing import connection
+
+from dependencies.python.db import layer_test
+from dependencies.python.db import get_ddb_connection
+import unittest
+from dependencies.python.db import Tag
+
+def test_layer_test():
+
+    ret = layer_test()
+    assert ret == "hello world"
+
+def test_get_ddb_connection():
+    connection = get_ddb_connection()
```

Comment:
This is my initial file which has been deleted. It wasn't deleted immediately, (sorry!), so that's probably why it showed up for you.  In the recent commit, this test is deleted so it shouldn't be there now for you. It is still  included in test_fridge.py where I had copied it from.